HLTrigger/Configuration/python/HLT_75e33/modules/hltTiclTracksterLinks_cfi.py

```diff
@@ -25,10 +19,22 @@
         type = cms.string('Skeletons'),
         wind = cms.double(0.036)
     ),
+    pluginInferenceAlgoTracksterInferenceByDNN = cms.PSet(
+      algo_verbosity = cms.int32(0),
+      onnxPIDModelPath = cms.FileInPath('RecoHGCal/TICL/data/RecoHGCal-TICL/ticlv5/onnx_models/linking/id_v0.onnx'),
+      onnxEnergyModelPath = cms.FileInPath('RecoHGCal/TICL/data/RecoHGCal-TICL/ticlv5/onnx_models/linking/energy_v0.onnx'),
+      eid_min_cluster_energy = cms.double(1),
+      eid_n_layers = cms.int32(50),
+      eid_n_clusters = cms.int32(10),
+      doPID = cms.int32(1),
+      doRegression = cms.int32(1),
+      type = cms.string('TracksterInferenceByDNN')
```

Comment:
For completeness, this PSet is missing the entries coming from
https://github.com/cms-sw/cmssw/blob/20b37d420e3d466539e652874237466089a05043/RecoHGCal/TICL/plugins/TracksterInferenceByDNN.cc#L140-L142